psi4/src/psi4/libscf_solver/rhf.cc

```diff
@@ -238,6 +238,10 @@ bool RHF::diis() { return diis_manager_->extrapolate(1, Fa_.get()); }
 void RHF::form_F() {
     Fa_->copy(H_);
     Fa_->add(G_);
+    if (!external_potentials_.empty()) {
```

Comment:
Any reason not to do a simple range-based for loop?
```C++
for (auto Vext : external_potentials_) {
    Fa_->add(Vext);
}
```

A bit cleaner IMO.

Reply:
Fixed.

Reply:
Looks like everything is still `for_each` based?

Reply:
I don't understand what happened. My local copy does have the range-based `for` loop...

Reply:
Looks like you just pushed that up. As a note I dont think you need the `if(empty)` call anymore.

Reply:
Cool, I removed that check.